flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -341,6 +343,19 @@ under the License.
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
 				<executions>
+					<execution>
+						<phase>package</phase>
```

Comment:
Shading `data-gen`  connector to flink-examples-streaming is necessary and it makes sense to me.  minor: We can add a meaningful <id> to describe the shade purpose